### PR TITLE
Update macos-shell-scripts.md

### PR DESCRIPTION
**Pull request by theneiljohnson**
Added comment to highlight that scripts will run after a reboot if a frequency is set.

   - **Run script as signed-in user**: Select **Yes** to run the script with the user's credentials on the device. Choose **No** (default) to run the script as the root user.

### CODE REVIEW

File: memdocs/intune/apps/macos-shell-scripts.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Ensure that the following prerequisites are met when composing shell scripts and
    - **Upload script**: Browse to the shell script. The script file must be less than 200 KB in size.
    - **Run script as signed-in user**: Select **Yes** to run the script with the user's credentials on the device. Choose **No** (default) to run the script as the root user.
    - **Hide script notifications on devices:** By default, script notifications are shown for each script that is run. End users see a *IT is configuring your computer* notification from Intune on macOS devices.
-   - **Script frequency:** Select how often the script is to be run. Choose **Not configured** (default) to run a script only once.
+   - **Script frequency:** Select how often the script is to be run. Choose **Not configured** (default) to run a script only once. Scripts with a frequency set will also run after a device reboot.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/macos-shell-scripts.md:70]:
```suggestion
   - **Script frequency:** Select how often the script is to be run. Choose **Not configured** (default) to run a script only once. Scripts with a frequency set will also run after a device restart.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @theneiljohnson, kindly commit the suggested change. Many thanks.

**Comment by theneiljohnson** [comment]
Added that scripts with a frequency set will also run on device restart.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and sign off this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
Thanks for the contribution! #sign-off

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**